[2023-01-27T16:26:16.419+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: Streaming_pipeline_meli.Status_info_mongodb manual__2023-01-27T16:26:14.836684+00:00 [queued]>
[2023-01-27T16:26:16.431+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: Streaming_pipeline_meli.Status_info_mongodb manual__2023-01-27T16:26:14.836684+00:00 [queued]>
[2023-01-27T16:26:16.431+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-27T16:26:16.431+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-01-27T16:26:16.431+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-27T16:26:16.447+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): Status_info_mongodb> on 2023-01-27 16:26:14.836684+00:00
[2023-01-27T16:26:16.453+0000] {standard_task_runner.py:55} INFO - Started process 695 to run task
[2023-01-27T16:26:16.458+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Streaming_pipeline_meli', 'Status_info_mongodb', 'manual__2023-01-27T16:26:14.836684+00:00', '--job-id', '1096', '--raw', '--subdir', 'DAGS_FOLDER/producer_dag.py', '--cfg-path', '/tmp/tmpjhb4oe1q']
[2023-01-27T16:26:16.459+0000] {standard_task_runner.py:83} INFO - Job 1096: Subtask Status_info_mongodb
[2023-01-27T16:26:16.513+0000] {task_command.py:389} INFO - Running <TaskInstance: Streaming_pipeline_meli.Status_info_mongodb manual__2023-01-27T16:26:14.836684+00:00 [running]> on host 502660defd2e
[2023-01-27T16:26:16.581+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Streaming_pipeline_meli
AIRFLOW_CTX_TASK_ID=Status_info_mongodb
AIRFLOW_CTX_EXECUTION_DATE=2023-01-27T16:26:14.836684+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-27T16:26:14.836684+00:00
[2023-01-27T16:26:46.621+0000] {producer_dag.py:70} ERROR - Error al conectar a mongoDB: 192.168.64.4:27017: timed out, Timeout: 30s, Topology Description: <TopologyDescription id: 63d3fb28e37a807a300592a3, topology_type: Single, servers: [<ServerDescription ('192.168.64.4', 27017) server_type: Unknown, rtt: None, error=NetworkTimeout('192.168.64.4:27017: timed out')>]>
[2023-01-27T16:26:46.622+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-27T16:26:46.656+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Streaming_pipeline_meli, task_id=Status_info_mongodb, execution_date=20230127T162614, start_date=20230127T162616, end_date=20230127T162646
[2023-01-27T16:26:46.722+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-27T16:26:46.748+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
